packages/mc-scripts/src/utils/get-config-diff.js

```diff
@@ -0,0 +1,246 @@
+const { red, green } = require('chalk');
+
+const notNestedKeySet = new Set(['name', 'description', 'url', 'icon']);
+
+const t = (tabs) => '\t'.repeat(tabs);
+
+const getStringDiff = (oldStr, newStr, key, tabs = 0) => {
```

Comment:
Nit: use clearer variable names.

```suggestion
const getStringDiff = (prevValue, nextValue, label, indentLevel = 0) => {
```